metron-interface/metron-rest/src/main/scripts/metron-rest.sh

```diff
@@ -36,6 +36,7 @@ METRON_SYSCONFIG="${METRON_SYSCONFIG:-/etc/default/metron}"
 METRON_LOG_DIR="${METRON_LOG_DIR:-/var/log/metron}"
 METRON_PID_FILE="${METRON_PID_FILE:-/var/run/metron/metron-rest.pid}"
 PARSER_CONTRIB=${PARSER_CONTRIB:-$METRON_HOME/parser_contrib}
+INDEXING_CONTRIB=${INDEXING_CONTRIB:-$METRON_HOME/indexing_contrib}
```

Comment:
Do we need to provide a default path for this?  Can we just require the user to define an environment variable "INDEXING_CONTRIB" if they want to do this (and not provide a default).

If there really is a need for us to define a default path, I am wondering if `$METRON_HOME/lib/contrib/indexing` fits a little better in our current hierarchy.

Reply:
Just to avoid the issue of 'what is a good default' and to avoid defining the same default in two different locations (REST and Indexing script), I would recommend just removing this line from the scripts and so not defining a default.

There is a file `/etc/default/metron` that gets laid down with a variety of different settings.  You can see that referred to here in line 35.  Eventually, that is probably where a value like this should be defined.  That would allow the MPack to set the value as needed and all scripts (indexing and REST) would be able to refer to the same value.  Of course, that is all future work.  I am not suggesting that we need Mpack changes now.

Reply:
@nickwallen This is consistent with what we've done for the parsers (see line 38 immediately above). We could possibly refactor, but I wouldn't advise it as part of this PR. Not defining a default would mean the responsibility is now on the end user for indexing, but not for parsers. I think that is going to be more confusing to a user in the current state. I do agree that these could probably be moved to the defaults script as a follow-on refactoring PR.

Reply:
That makes sense to me @mmiklavc.  Definitely a follow-on task.